### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by pinglin**
Because

- we are continuously improving the documentation

This commit

- update README.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,7 +63,7 @@ prune:			## Remove all services containers and system prune everything
 	@docker system prune -f --volumes
 .PHONY: prune
 
-build:			## Build local docker image
+build:			## Build instill/vdp:dev local development image
```
Comment (xiaofei-du) [inline Makefile:66]:
```suggestion
build:			## Build instill/vdp:dev for local development
```

File: README.md [hunk 2]
```diff
@@ -56,14 +55,15 @@ We use **data connector** as a general term to represent data source and data de
...
-$ cd vdp
+# Build instill/vdp:dev local development image
+$ make build
 
-# Note that this may take a while due to the enormous size of the Triton server image,
-# but the image pulling process should be just a one-time effort.
+# Launch all services.
 $ make all
 ```
+:warning: The download of the Triton server image will take a while, but the image pulling process should be just a one-time effort.
```
Comment (xiaofei-du) [inline README.md:66]:
```suggestion
:warning: Downloading the Triton server image will take a while, but it should be just a one-time effort.
```

File: README.md [hunk 3]
```diff
@@ -56,14 +55,15 @@ We use **data connector** as a general term to represent data source and data de
 Execute the following commands to start pre-built images with all the dependencies:
 
 ```bash
-$ git clone https://github.com/instill-ai/vdp.git
+$ git clone https://github.com/instill-ai/vdp.git && cd vdp
 
-$ cd vdp
+# Build instill/vdp:dev local development image
+$ make build
```
Comment (xiaofei-du) [inline README.md:61]: Do we need to add `make build` for Quickstart? I thought `make all` will cover it?

**Review by xiaofei-du — CHANGES_REQUESTED** [review]

**Comment by xiaofei-du** [comment]
We can change
```markdown
The core concept of VDP is _pipeline_. A pipeline is an end-to-end workflow that automates end-to-end visual data processing. Each pipeline consists of three ordered components:
``` 
to
```
The core concept of VDP is _pipeline_. A pipeline is an end-to-end workflow that automates a sequence of tasks to process visual data. Each pipeline consists of three ordered components:
```

File: README.md (hunk 3 above)
Comment (pinglin) [reply README.md:61]: We probably don't want to build `instill-ai/vdp:dev` everytime when `make all` so `make build` is a step alone now.

**Review by xiaofei-du — APPROVED** [review]
LGTM 👍

**Pull request merged**